src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -1114,7 +1120,7 @@ void Replayer<I>::handle_copy_image(int r) {
     if (m_perf_counters) {
       m_perf_counters->inc(l_rbd_mirror_snapshot_replay_bytes, m_snapshot_bytes);
       m_perf_counters->inc(l_rbd_mirror_snapshot_replay_snapshots);
-      m_perf_counters->tinc(l_rbd_mirror_snapshot_replay_snapshots_time, time);
+      m_perf_counters->tinc(l_rbd_mirror_snapshot_replay_snapshots_time, duration);
```

Comment:
As it is, this doesn't compile: a similar time -> duration is missing a few lines above.